### PR TITLE
Annotate reports with the platform on which they are running.

### PR DESCRIPTION
**Pull request by hellais**

### CODE REVIEW

File: ooni/settings.py [hunk 1]
```diff
@@ -325,6 +326,20 @@ def web_ui_url(self):
...
+    def platform(self):
+        if os.path.exists('/etc/default/lepidopter'):
+            return 'lepidopter'
+        system = platform.system()
+        if system == 'Darwin':
+            return 'macos'
+        elif system == 'Linux':
+            return 'linux'
+        elif system == 'Windows':
+            # Really?
```
Comment (bassosimone) [inline ooni/settings.py]: :-D

**Comment by bassosimone** [comment]
Looks good to me!

**Comment by bassosimone** [comment]
Yes, I think this is a very good thing to do! How would this end up into the report? Just to know it and do something equivalent for measurement-kit and possibly also at least add an issue to specify this information into the specification of our data format.

File: ooni/settings.py [hunk 2]
```diff
@@ -325,6 +326,20 @@ def web_ui_url(self):
...
+            return 'windows'
+        return 'unknown'
```
Comment (darkk) [inline ooni/settings.py]: Why don't you want to return `platform.system().lower()` instead of `'unknown'`?

**Pull request merged**